### ISSUE TITLE
CSS file is not available in the last package

### ISSUE THREAD

**Issue by jesuismaxime** [ownego/polaris-vue#186]
As the documentation state, i used `import '@ownego/polaris-vue/dist/styles.css'` to import the CSS into my `main.js` file, but it results in this compilation error:

```
Failed to resolve import "@ownego/polaris-vue/dist/styles.css"
```

**Comment by juzser** [comment]
@jesuismaxime You can check in your `node_modules` directory to see if there is css file or not. 
I guess this issue happens because you have alias `@` or `node_modules` as different path in Vite config.

**Comment by jesuismaxime** [comment]
There's no CSS file in the package except a `polaris.css` in the `/lib` directory – which is not exported so cannot be used. In fact there's no `/dist` at all in the node_modules directory, event VSCode is not able to complete any import path.

Note: I used `@` alias on every project and that never causes a conflit, whether it's a npm package or github package. Although, I changed my `@` alias to something else to be 100% certain that this doesn't cause the error.

In the mean time, I copied the CSS file from the Storybook doc to be able to test out this library.
